sdks/builds/llvm.mk

```diff
@@ -32,9 +32,18 @@ _llvm_$(1)_CONFIGURE_FLAGS= \
 	--enable-targets="arm,aarch64,x86" \
 	$$(if $$(filter $$(UNAME),Darwin),--enable-libcpp)
 
+_llvm_$(1)_CMAKE_FLAGS = \
+	-DCMAKE_INSTALL_PREFIX=$$(TOP)/sdks/out/llvm-$(1) \
+	-DCMAKE_BUILD_TYPE=Release \
+	-DLLVM_TARGETS_TO_BUILD="X86;ARM;AArch64" \
+	-G Ninja \
+	$(LLVM_SRC)
```

Comment:
We need to be able to build 32 and 64 bits versions.

Reply:
It also support building with cmake.

Reply:
I have just tried with https://github.com/mono/llvm/commit/bdb3a116dbf4a77ef5f6f70d7f845bdaca2b24ca and using cmake just works.